Makefile

```diff
@@ -104,9 +104,12 @@ docker:
 run-docker:
 	@docker run \
 	    -it --rm \
+	    -e "CHECKMATE_API_KEY=dummy-key" \
+	    -e "CHECKMATE_URL=https://qa-checkmate.hypothes.is" \
```

Comment:
These are wrong values, but they allow the app to at least start. If there is a better option, let me know.